### DISCUSSION TITLE
【自我照顧 Step1】

### DISCUSSION THREAD

**Discussion by Chienyuuu6** [1101-GS4719-C/Python-Community-GS4719-C#1357] [General]
#MentalHealthAwarenessMonth
我們總是關注外在的人事物，總是羨慕、欣賞社群媒體上的其他人看似發光的生活，Why not follow “myself”?

如何自我照顧？
Step 1 : 認識自己🧚🏼‍♀️
✨最近的狀況：表層情緒&深層情緒
要對自己誠實！（面對對深層情緒）
看得清、理得清才有好好照顧它的機會！

✨最近情緒的改變
覺得生氣？變得沒耐心？
對自己沒信心？很想哭？感到不安？

✨最近思考上的變化
記憶力變差？負面思考？
決定的能力下降（面對微小瑣事猶豫）
攻擊性變強？沒有希望感？

✨最近生理上的變化
沒有理由的疲累感？失眠or過眠？
飲食改變（食慾不佳？）身體不適？
無法專注？有疼痛感？

歡迎大家在這個板上面分享自己在這個Step1所察覺到的自己～
一年來我經過這些分析對自己有更深的認識！我覺得對生活上感到emo的時候是很快速能夠釐清自我的方法！

**Comment by TW-Maverick** [comment]
1. 對自己未來的工作跟升學感到擔憂、被自己的身體搞了快三個禮拜🤦‍♂️
2. 我一直對周遭人的一舉一動很敏感，很容易在意別人
3. 通常都負面思考，但會一直體醒自己要正向一點👌
4. 因為感冒，咳了快兩個禮拜，晚上還有失眠的問題QQ

感謝你分享【自我照顧】的4步驟💯
